src/mono/sample/wasm/browser-advanced/index.html

```diff
@@ -7,18 +7,17 @@
   <title>Wasm Browser Advanced Sample</title>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="modulepreload" href="./main.js" />
-  <link rel="modulepreload" href="./dotnet.js" />
+  <script type='module' src="./main.js"></script>
+  <script type='module' src="./dotnet.js"></script>
   <link rel="preload" href="./mono-config.json" as="fetch" crossorigin="anonymous">
-  <link rel="preload" href="./dotnet.wasm" as="fetch" crossorigin="anonymous">
-  <link rel="preload" href="./icudt.dat" as="fetch" crossorigin="anonymous">
-  <link rel="preload" href="./managed/System.Private.CoreLib.dll" as="fetch" crossorigin="anonymous">
+  <link rel="prefetch" href="./dotnet.wasm" as="fetch" crossorigin="anonymous">
+  <link rel="prefetch" href="./icudt.dat" as="fetch" crossorigin="anonymous">
+  <link rel="prefetch" href="./managed/System.Private.CoreLib.dll" as="fetch" crossorigin="anonymous">
```

Comment:
Shouldn't the link element end with `/>`?

Reply:
It's optional in html5